### DISCUSSION TITLE
Training dataset location

### DISCUSSION THREAD

**Discussion by ednalibini** [PaulJoshi/Gestify#8] [Q&A] [answered]
Where to locate the training dataset used for this project in the repository?

**Comment by PaulJoshi** [comment]
We have not pushed the training dataset along with the repo.

**Comment by parvathiAmbareesh** [comment]
Use [ImageCollection.ipynb](https://github.com/PaulJoshi/Gestify/blob/main/Image%20Collection.ipynb) or [ImageCollection.py](https://github.com/PaulJoshi/Gestify/blob/main/ImageCollection.py) in root to make your own training dataset. Dataset will be stored by default in /Tensorflow/Images.

**Comment by peterjohnson1000** [comment, marked as answer]
We did not push our training dataset due to privacy concerns. However, you can use [ImageCollection.ipynb](https://github.com/PaulJoshi/Gestify/blob/main/Image%20Collection.ipynb) or [ImageCollection.py](https://github.com/PaulJoshi/Gestify/blob/main/ImageCollection.py) in root to make your own training dataset. Dataset will be stored by default in /Tensorflow/Images.
